test/keyv.js

```diff
@@ -75,6 +75,44 @@ test.serial('.set(key, val, ttl) where ttl is "0" overwrites default tll option
 	tk.reset();
 });
 
+test.serial('.get(key) gets only the value', async t => {
```

Comment:
You don't need to test this, it's tested in [`@keyv/test-suit`](https://github.com/lukechilds/keyv-test-suite)